Resources/views/CategoryAdmin/tree.html.twig

```diff
@@ -23,7 +23,13 @@ file that was distributed with this source code.
             <li class="sonata-ba-list-field" objectId="{{ element.id }}" >
                 <div class="sonata-tree__item"{% if depth < 2 %} data-treeview-toggled{% endif %}>
                     {% if element.parent or root %}<i class="fa fa-caret-right" data-treeview-toggler></i>{% endif %}
-                    <a class="sonata-tree__item__edit" href="{{ admin.generateObjectUrl('edit', element) }}">{{ element.name }}</a>
+                    {% if admin.hasRoute('edit') %}
```

Comment:
Basically also added access check.  Access check by itself does not check if the route exist. If I use only `hasAccess` method still throws exception.